web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\dpl_react_apps\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Controller for rendering full page DPL React apps.
+ */
+class DplReactAppsController extends ControllerBase {
+
+  /**
+   * Render search result app.
+   *
+   * @return mixed[]
+   *   Render array.
+   */
+  public function search(): array {
+    return [
+      'search-result' => dpl_react_render('search-result'),
+    ];
+  }
+
+  /**
+   * Render material page.
+   */
+
+  /**
+   * Render material page.
+   *
+   * @param string $pid
+   *   A material post id.
+   *
+   * @return mixed[]
+   *   Render array.
+   */
+  public function material(string $pid): array {
+    // Translation context.
+    $c = ['context' => 'Material Page'];
+
+    return [
+      'material' => dpl_react_render('material', [
+        'pid' => $pid,
+        'material-header-author-by-text' => $this->t('By', [], $c),
+        'periodikum-select-year-text' => $this->t('Year', [], $c),
+        'periodikum-select-week-text' => $this->t('Week', [], $c),
+        'reserve-book-text' => $this->t('Reserve Book', [], $c),
+        'fine-on-bookshelf-text' => $this->t('Find at Book Shelf', [], $c),
```

Comment:
```suggestion
        'reserve-book-text' => $this->t('Reserve book', [], $c),
        'fine-on-bookshelf-text' => $this->t('Find at book Shelf', [], $c),
```

Minor capitalization changes.